scripts/tools/generate_esp32_chip_factory_bin.py

```diff
@@ -575,18 +614,31 @@ def any_base_int(s): return int(s, 0)
 
     populate_factory_data(args, spake2p_params)
 
-    if args.dac_key:
+    if args.dac_key and not args.dac_in_secure_cert:
         gen_raw_ec_keypair_from_der(args.dac_key, FACTORY_DATA['dac-pub-key']['value'], FACTORY_DATA['dac-key']['value'])
+    if args.dac_key:
+        gen_raw_ec_keypair_from_der(args.dac_key, os.path.abspath('dac_raw_pubkey.bin'), os.path.abspath('dac_raw_privkey.bin'))
```

Comment:
This is not required, public key is extracted from the certificate and the private key goes into secure cert partition.